jvb/src/main/kotlin/org/jitsi/videobridge/message/BridgeChannelMessage.kt

```diff
@@ -241,10 +259,12 @@ class ReceiverVideoConstraintMessage(val maxFrameHeight: Int) : BridgeChannelMes
 /**
  * A message sent from the bridge to a client, indicating that the dominant speaker in the conference changed.
  */
+@JsonInclude(JsonInclude.Include.NON_NULL)
 class DominantSpeakerMessage @JvmOverloads constructor(
-    var dominantSpeakerEndpoint: String,
+    dominantSpeakerEndpoint: String,
```

Comment:
Does `dominantSpeakerEndpoint` actually need to be a `var`?  I don't see it being set anywhere, other than `BridgeChannelMessageTest`.

Reply:
It's just the test. I can remove it